presto-spi/pom.xml

```diff
@@ -23,6 +23,11 @@
         </dependency>
 
         <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>units</artifactId>
```

Comment:
You need to update PluginManager to include io.airlift.units to the "parent-first" classes.